### ISSUE TITLE
load countries on modules PhoneInput

### ISSUE THREAD

**Issue by RahavLussato** [catamphetamine/react-phone-number-input#151]
I have issues with the modules code.
its looks like somehow i'm not loading the countries as is should be.
the source code looks good but the commonjs and the modules are not working for me.

import countries code:
```
var _countries = __webpack_require__(1398);
```

defualtprops code:
```
countries: _countries.countries,
```

and for some reason, `_countries` holds the array and not _countries.countries which is undefined.

<img width="402" alt="screen shot 2018-06-28 at 15 38 36" src="https://user-images.githubusercontent.com/6713560/42034821-e77415cc-7ae9-11e8-9e98-afb0c1f4d249.png">

i have '.json' as part of the resolve extensions and i also tries to use json-loader (went the same way).

using webpack 3.8.1

**Comment by catamphetamine** [comment]
> its looks like somehow i'm not loading the countries as is should be.

Clarify yourself.
Are you observing some kind of bugs?

**Comment by RahavLussato** [comment]
@catamphetamine yes i get a lot of undefined issues because in the code it always tries to get the default counteris as _countries.countries which is undefined

**Comment by catamphetamine** [comment]
@RahavLussato I still don't understand what your exact issue is.
You can try to debug it yourself using `npm build` command.

**Comment by RahavLussato** [comment]
@catamphetamine the exact issue is when i use it like that:
```
<PhoneInput
                placeholder="Enter phone number"
                value={ value }
                onChange={ value => handlePhoneChange(value)}
                error={ value ? (isValidNumber(value) ? undefined : 'Invalid phone number') : 'Phone number required' }
                indicateInvalid/>
```

i get the following error:
"input-control.js:47 Uncaught TypeError: Cannot read property 'indexOf' of undefined"

after debugging that i saw that for some reason the countries array is under `_countries` and not under `_countries.countries` as the code expects it to be.

and my question is what am i missing here ?

**Comment by catamphetamine** [comment]
On which line exactly is the error thrown?

**Comment by catamphetamine** [comment]
Ok, I think I can see the issue: it prefers `countries.json` instead of `contries.js` for some reason.

**Comment by RahavLussato** [comment]
on modules/input-control line 47
```
if (countries.indexOf(country) < 0) {
```
countries is undefined because on modules/PhoneInput.js on line 527
```
countries: _countries.countries,
```

**Comment by catamphetamine** [comment]
Ok, I'll rewrite `countries.json` and release a new version

**Comment by RahavLussato** [comment]
@catamphetamine thanks i'll update once you release it

**Comment by RahavLussato** [comment]
@catamphetamine you can just rename `countries.json` to something like `countries_data_.json` and change it on the imports, worked foe me.

**Comment by catamphetamine** [comment]
Released version `1.1.9`.

**Comment by RahavLussato** [comment]
@catamphetamine thanks its working
